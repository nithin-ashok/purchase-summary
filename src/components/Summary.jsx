import React from "react";

const Summary = ({children}) => (
  <div className="center summary">
      {children}
  </div>
);

export default Summary;