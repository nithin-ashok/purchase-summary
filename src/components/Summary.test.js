import React from 'react';
import Summary from "./Summary";
import renderer from 'react-test-renderer';

it("renderes correctly Summary component", () => {
    const renderSummary = jest.fn();
    const tree = renderer.
    create(<Summary>
        {renderSummary()}
    </Summary>).toJSON();

    expect(tree).toMatchSnapshot();
});