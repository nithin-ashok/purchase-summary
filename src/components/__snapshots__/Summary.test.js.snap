// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderes correctly Summary component 1`] = `
<div
  className="center summary"
/>
`;
